airtime_mvc/application/configs/conf.php

```diff
@@ -32,6 +32,7 @@ public static function loadConfig() {
         $CC_CONFIG['stationId'] = $values['general']['station_id'];
         $CC_CONFIG['phpDir'] = $values['general']['airtime_dir'];
         $CC_CONFIG['forceSSL'] = isset($values['general']['force_ssl']) ? $values['general']['force_ssl'] : FALSE;
```

Comment:
I would reconsider this line, and how it is then used as value in some conditions.
What I understand is that, if anything is set, say `no` (or `yes`, doesn't matter), the `$forceSSL` var will always evaluate as true, and only to false if it isn't set.

This is not the behavior I saw in the python code.

Reply:
I don't know PHP well enough to fix this...

Reply:
I'll try to push something

Reply:
See https://github.com/paddatrapper/libretime/pull/14

Reply:
Merged into this PR